### PR TITLE
Gourmonger Meal Priority Fix

### PR DESCRIPTION
**Pull request by GreatBigFailure**
tl;dr - Gourmongers now have random target priority and interrupt a meal if it's eaten first which fixes everything. Also simple mobs are worth twice as much nutrition as before.

After watching this setup for a while 
verypowerfulsetup
and witnessing the gourmongers starve constantly despite the pile of food I realized something: "Only one is splitting. It's eating all the food, all of it." So I looked a bit deeper and turns out mob targeting isn't as random as I thought. Turns out they literally just pick the first thing in a list of priorities. 
All your gourmonger setups were failing because they were all competing for the same donut. The one who started eating first would finish first and get the whole meal while the others starved, then he would start on the next donut first as well and the cycle repeats. 

So now they target something random from their list of highest priorities. They also stop attempting to finish a meal that was already eaten. Additionally I freed them from the chains of fuzzyrules which should make them more likely to ditch a human target if they see a pizza while loose, maybe.  Finally a little unatomic tweak: simple mobs used to be worth their max health in nutrition, now they're worth twice that because I promised.

I now wash my hands of all gourloosing. 
[hotfix]

:cl:
 * bugfix: Gourmongers no longer create a cycle of hunger by all trying to eat the same donut at once

### CODE REVIEW

**Comment by Eneocho** [comment]
They loosed despite the suppression fields. I know you want them to be dangerous and shit, but they shouldn't be able to loose through fields.

**Comment by ShiftyRail** [comment]
A better way to handle this would be to keep fuzzy rules but to add a harsh penalty to a meal being already targetted by another mob. Alternatively, have them fight for a meal if that's what you want.

As for the "ditch a target" thing, I don't know if it has anything to do with fuzzy rules, actually. You would need to make them re-evaluate their targets list periodically.
